README_developers.md

```diff
@@ -29,3 +29,17 @@ Note: Currently, only Vijay and Arif can do this, as they are the only ones with
 python -m build
 twine upload dist/*
 ```
+## Create Conda recipe
+**NOTE:** This is needed to be done only once to add the recipe to conda-forge/staged-recipe.
+To update the conda release follow conda release section below.
```

Comment:
Where is this section? Not sure I see it...
Isn't the idea that this does not need to be done for each new pypi release? This sentence seems to say the opposite.

Reply:
added the instructions to upgrade to new version